stdlib/InteractiveUtils/src/InteractiveUtils.jl

```diff
@@ -92,6 +92,12 @@ function versioninfo(io::IO=stdout; verbose::Bool=false)
     if !isempty(Base.GIT_VERSION_INFO.commit_short)
         println(io, "Commit $(Base.GIT_VERSION_INFO.commit_short) ($(Base.GIT_VERSION_INFO.date_string))")
     end
+    if Base.GIT_VERSION_INFO.tagged_commit
+        # If this is a tagged commit, print the tagged release banner.
+        # For example, for official release binaries, the tagged release banner is
+        # something like "Official https://julialang.org/ release".
```

Comment:
I think this is actually a bit misleading, since it might not actually be an official release but rather a source build of a tagged commit. You could add to the commit information above something like
```julia
Base.GIT_VERSION_INFO.tagged_commit ? "tagged version" : ""
```

Reply:
Hmmm. In that case (source build of a tagged commit), what will the value of `Base.TAGGED_RELEASE_BANNER` be?

Reply:
Should be empty unless the user specified it manually when building. But yeah, thinking more about this, it seems pretty unlikely that somebody would actually do that, so what you have is probably fine.

Reply:
Hmmm. It would be confusing if someone set the release banner when they built it themselves from source.

So it's probably still best for us to ask people specifically, "how did you download and install Julia."